src/components/Toggle/Toggle.scss

```diff
@@ -0,0 +1,61 @@
+@import "../../styles/states.scss";
+@import "../../styles/global-css-settings.scss";
+
+@mixin toggle-animation {
+  transition: 100ms $expand-animation-timing;
+}
+
+.monday-style-toggle{
+  &__wrapper {
+    display: flex;
+    align-items: center;
+  }
+
+  &__toggle {
+    @include reset-button();
+    @include toggle-animation;
+    margin-inline: $spacing-small;
```

Comment:
![image](https://user-images.githubusercontent.com/2057934/120966427-e52a0b80-c76e-11eb-9d75-1a1da6ec172a.png)
I think it's not supported in edge which I think we're supposed to support?
https://caniuse.com/?search=margin-inline

Reply:
yes this is not supported :( i have to change it :(